practice/practice-02/src/nav-menu.pug

```diff
@@ -0,0 +1,8 @@
+nav.nav
```

Comment:
For all partial content like this, could we have a folder

Reply:
Fixed.